h/static/styles/partials-v2/_share-widget.scss

```diff
@@ -0,0 +1,138 @@
+.share-widget-template {
+  visibility: hidden;
+  // we want template to have no effect
+  // on the page layout, but keep it in it's
+  // current flow
+  height:0;
+  width:0;
+}
+
+.share-widget.is-open {
+  display: flex;
+  flex-direction: column;
+}
+
+.share-widget {
+  position: absolute;
+  top:0;
+  z-index: 1;
+  background: $white;
+  border: 1px solid $grey-3;
+  border-radius: 2px;
+  width: 200px;
+  font-size: $normal-font-size;
+  cursor: default;
+  box-shadow: 0px 0px 4px 0px rgba(0,0,0,0.15);
+
+  // Styling for the dialog box tail(arrow) pointing down at the
+  // 'Share' button
+  &:after, &:before {
+    top: 100%;
+    right: 10px;
+    border: solid transparent;
+    content: " ";
+    height: 0;
+    width: 0;
+    position: absolute;
+    pointer-events: none;
+  }
+
+  &:after {
+    border-color: rgba(255, 255, 255, 0);
+    border-top-color: $white;
+    border-width: 5px;
+    margin-right: -5px;
+  }
+
+  &:before {
+    border-color: rgba(211, 211, 211, 0);
+    border-top-color: $grey-3;
+    border-width: 6px;
+    margin-right: -6px;
+  }
+}
+
+.share-widget-msg {
+  color: $grey-5;
+  margin: -5px 10px 10px 10px;
+  line-height: 15px;
+  font-size: 11px;
+}
+
+.share-widget-msg__audience {
+  font-style: italic;
+}
+
+.share-widget-link {
+  position: relative;
+
+  font-size: $normal-font-size;
+  margin: 10px;
+  padding: 5px;
+  color: $grey-5;
+  border: 1px solid $grey-3;
+  border-radius: 2px;
+  background: $grey-1;
+  white-space: nowrap;
+  overflow: hidden;
+  display: flex;
+  flex-direction: row;
+}
+
+.share-widget-link:hover {
+  color: $grey-7;
+}
+
+.share-widget-link__text {
+  overflow: hidden;
+  border: none;
+  width: 100%;
+  background: $white;
+}
+
+.share-widget-link__btn {
+  padding: 0px;
+  margin-left: 7px;
+  border: 0;
+  background: transparent;
+}
+
+.share-widget-link__btn--copy {
+  margin-top: 5px;
+  margin-bottom: 5px;
+  width: 13px;
+  height: 16px;
+}
+
+.share-widget-action * {
+  fill: $grey-4;
```

Comment:
I'm afraid this isn't documented property in the comments for the `svg_icon` helper, but the intent (and how our other SVG icons work) is that you set a `color` property on the containing `<svg>` element and then use `fill="currentColor"` or `stroke="currentColor"` in the SVG itself to apply that color.